src/js/utils/formatting/build-matcher.js

```diff
@@ -7,5 +7,5 @@ import searchSanitize from 'js/utils/formatting/search-sanitize';
 export default memoize(function(query) {
   const searchWords = map(words(searchSanitize(query)), RegExp.escape);
 
-  return new RegExp(`\\b${ searchWords.join('|') }`, 'gi');
+  return new RegExp(searchWords.join('|'), 'gi');
```

Comment:
The failing test is for the team component.

<img width="240" height="226" alt="Screenshot 2025-08-14 at 4 30 43 PM" src="https://github.com/user-attachments/assets/dab5d389-c4e7-4413-bfec-56f052c9ffb3" />

This is returning more results now. 4 instead of 2.

I can update the test for this to pass. But is this what we want? These currently only match the beginning of the name in the team/owner components.

Reply:
If we only want to match the beginning of the names, we could add a function parameter to `build-matcher.js` that uses a different regular expression to include the `\b` or not.

Reply:
oh yep.. we should probably add a second utlity for just the patient search for now